lib/src/pages/layouts/yaru_landscape_layout.dart

```diff
@@ -63,51 +104,141 @@ class _YaruLandscapeLayoutState extends State<YaruLandscapeLayout> {
-                      ? widget.pageBuilder(context, _selectedIndex)
-                      : widget.pageBuilder(context, 0),
-                ),
-              ],
-              onPopPage: (route, result) => route.didPop(result),
-              observers: [HeroController()],
-            ),
+      ),
+      child: Scaffold(
+        appBar: widget.appBar,
+        body: YaruMasterListView(
+          length: widget.length,
+          selectedIndex: _selectedIndex,
+          onTap: _onTap,
+          builder: widget.tileBuilder,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildPage(BuildContext context) {
+    final theme = YaruMasterDetailTheme.of(context);
+
+    return Theme(
+      data: Theme.of(context).copyWith(
+        pageTransitionsTheme: theme.landscapeTransitions,
+      ),
+      child: Navigator(
+        pages: [
+          MaterialPage(
+            key: ValueKey(_selectedIndex),
+            child: widget.length > _selectedIndex
+                ? widget.pageBuilder(context, _selectedIndex)
+                : widget.pageBuilder(context, 0),
+          ),
+        ],
+        onPopPage: (route, result) => route.didPop(result),
+        observers: [HeroController()],
+      ),
+    );
+  }
+
+  Widget _buildLeftPaneResizer(
+    BuildContext context,
+    BoxConstraints boxConstraints,
+  ) {
+    return Positioned(
+      child: AnimatedContainer(
+        duration: _kLeftPaneResizingRegionAnimationDuration,
+        color:
+            _isHovering || _isDragging ? _separatorColor : Colors.transparent,
+        child: MouseRegion(
+          cursor: SystemMouseCursors.resizeColumn,
+          onEnter: (event) => setState(() {
+            _isHovering = true;
+          }),
+          onExit: (event) => setState(() {
+            _isHovering = false;
+          }),
+          child: GestureDetector(
+            onPanStart: (details) => setState(() {
+              _isDragging = true;
+              _initialPaneWidth = _leftPaneWidth;
+            }),
+            onPanUpdate: (details) => setState(() {
+              _paneWidthMove += details.delta.dx;
+              final width = _initialPaneWidth + _paneWidthMove;
+              final maxWidth = boxConstraints.maxWidth - widget.pageMinWidth;
+
+              final previousPaneWidth = _leftPaneWidth;
+
+              if (width >= maxWidth) {
+                _leftPaneWidth = maxWidth;
+              } else if (width < widget.leftPaneMinWidth) {
+                _leftPaneWidth = widget.leftPaneMinWidth;
+              } else {
+                _leftPaneWidth = width;
+              }
+
+              if (previousPaneWidth != _leftPaneWidth) {
+                widget.onLeftPaneWidthChange?.call(width);
+              }
+            }),
+            onPanEnd: (details) => setState(() {
+              _isDragging = false;
+              _paneWidthMove = 0.0;
+            }),
           ),
         ),
-      ],
+      ),
+      height: MediaQuery.of(context).size.height,
```

Comment:
`Positioned.fill` is that I was searching, thanks :)

Reply:
I found another solution, because `Positioned.fill` doesn't have width/height:

```dart
Positioned(
  width: _kLeftPaneResizingRegionWidth,
  top: 0,
  left: 0,
  bottom: 0,
)
```

I don't know why I did though about it before 😅